jerry-core/debugger/debugger.c

```diff
@@ -514,6 +519,14 @@ jerry_debugger_breakpoint_hit (uint8_t message_type) /**< message type */
 
   while (!jerry_debugger_receive ())
   {
+    /* timeout */
+    #ifdef __unix__
+    syscall (SYS_nanosleep, &(const struct timespec)
+    {
+        0, 100  /*second, millisecond*/
```

Comment:
The comment is misleading is many ways. First of all, the `timespec` struct has fields for seconds and nanoseconds. Or, 100 nanoseconds is not 1 millisec but 0.1 microsec.